Cargo.toml

```diff
@@ -1,39 +1,41 @@
 [package]
-name = "curv"
-version = "0.7.1"
+name = "curv-kzen"
+version = "0.8.0-rc1"
```

Comment:
can you explain the name change ? 
why  rc1?

Reply:
We publish crate on crates.io under this name. I had to change the crate name to make automatic publishing possible (when you tag commit, and a new version is being uploaded to crates.io). But that's a package name, [library name remains the same](https://github.com/ZenGo-X/curv/blob/2a75890edd23096f3adf34d77e72b99f1fd629ba/Cargo.toml#L11).

About the version — rc1 stands for "release candidate 1". It means that it's something close to what will be released in next version. My plan — I want to release curv v0.8.0-rc1, and then gradually update others crates. Updating other crates might require some changes in `curv`, and I'll be able to make these changes and publish `v0.8.0-rc2`, `v0.8.0-rc3`, etc. without messing with semver policies. Eventually, after all downstream crates are updated, `v0.8.0` will be released.